libs/ui/src/search_select.tsx

```diff
@@ -69,9 +76,12 @@ interface SearchSelectBaseProps<T extends string = string> {
   options: Array<SelectOption<T>>;
   'aria-label'?: string;
   style?: React.CSSProperties;
-  placeholder?: string;
+  placeholder?: React.ReactNode | string;
```

Comment:
`React.ReactNode` is a union that includes `string`.
```suggestion
  placeholder?: React.ReactNode;
```